### DISCUSSION TITLE
Problem with the reversing throttle

### DISCUSSION THREAD

**Discussion by jachu1978** [flybywiresim/aircraft#4690] [Help] [answered]
Hello, I would like to ask for help in setting the reversing throttle, I had a good setting for some time and now I have a problem

[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2,0
DetentReverseFull = -1,00
DetentReverseIdle = -0,70
DetentIdle = -0,44
DetentClimb =
-0,10 DetentFlexMct = 0,53
DetentTakeOffGoAround = 1,00

**Comment by ExampleWasTaken** [comment, marked as answer]
Try to replace the commas `,` with decimal points `.`.

**Reply by aguther** [reply]
this settings format is outdated. use once the efb to configure and save then you get the new format and you can edit it yourself in the file if you want. but keep in mind to have enough dead zone for each detent.
